### ISSUE TITLE
Accessibility View: accessibility.verbosity.editor is not a registered configuration

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#186302]
Testing #186214

1. Open the accessibility view from Copilot Chat
2. Press "D" to disable accessibility verbosity
3. Notice error notification `Unable to write to User Settings because accessibility.verbosity.editor is not a registered configuration.`

I noticed that the `accessibility.verbosity.panelChat` still gets nicely written.

![Screenshot 2023-06-27 at 11 55 46](https://github.com/microsoft/vscode/assets/1926584/4d50f96d-ae5f-4ad2-a074-4ad37716e527)

**Comment by meganrogge** [comment]
Good catch, we don't have such a setting at the moment.
